geode-client-native/src/cppcache/impl/CppCacheLibrary.cpp

```diff
@@ -79,7 +84,6 @@ typedef ACE_Singleton<CppCacheLibrary,ACE_Recursive_Thread_Mutex> TheLibrary;
 // internals are prepared. fw_helper framework will handle this.
 CppCacheLibrary* CppCacheLibrary::initLib( void )
 {
-  ACE::init();
```

Comment:
Won't removing the ACE::init() call affect all existing users?  That is, they would need to modify their applications to add the invocation directly.  I understand this is important for your use case.  Is there a way you could conditionally skip the library initialization rather than remove it entirely?